test/test_fx.py

```diff
@@ -15,6 +15,21 @@
 skipIfNoTorchVision = unittest.skipIf(not HAS_TORCHVISION, "no torchvision")
 
 class TestFX(TestCase):
+    def checkGraphModule(self, m: torch.nn.Module, args, kwargs=None):
+        """Check that an nn.Module's results match the GraphModule version
+        for a given set of args/kwargs.
+        """
+        if kwargs is not None:
```

Comment:
Can we just do `kwargs = kwargs if kwargs else {}` above?

Reply:
sure, #python